osu.Game.Tests/Visual/Gameplay/TestSceneSkinnableHealthDisplay.cs

```diff
@@ -38,23 +43,18 @@ public void TestHealthDisplayIncrementing()
         {
             AddRepeatStep(@"decrease hp", delegate
             {
-                foreach (var healthDisplay in healthDisplays)
-                    healthDisplay.Current.Value -= 0.08f;
+                healthProcessor.Health.Value = 0.08f;
             }, 10);
 
             AddRepeatStep(@"increase hp without flash", delegate
             {
-                foreach (var healthDisplay in healthDisplays)
-                    healthDisplay.Current.Value += 0.1f;
+                healthProcessor.Health.Value = 0.1f;
             }, 3);
 
             AddRepeatStep(@"increase hp with flash", delegate
             {
-                foreach (var healthDisplay in healthDisplays)
-                {
-                    healthDisplay.Current.Value += 0.1f;
-                    healthDisplay.Flash(new JudgementResult(null, new OsuJudgement()));
-                }
+                healthProcessor.Health.Value = 0.1f;
+                healthProcessor.ApplyResult(new JudgementResult(new HitCircle(), new OsuJudgement()));
```

Comment:
needs to satisfy the conditions in health display (currently doesn't satisfy `judgement.IsHit`):
https://github.com/ppy/osu/blob/755588258e90883f22a89cee219d6e44373b5887/osu.Game/Screens/Play/HUD/HealthDisplay.cs#L42

```suggestion
                healthProcessor.ApplyResult(new JudgementResult(new HitCircle(), new OsuJudgement())
                {
                    Type = HitResult.Perfect,
                });
```